pkg/apis/events/v1alpha1/scheduler_types.go

```diff
@@ -38,7 +39,12 @@ type Scheduler struct {
 	Spec   SchedulerSpec   `json:"spec"`
 	Status SchedulerStatus `json:"status"`
 }
-
+const (
+	// Scheduler CloudEvent type
+	SchedulerEventType = "com.google.cloud.sheduler.job.execute"
+	//Attributes jobName key
+	JobName = "jobName"
```

Comment:
does this needs to be here? Can it just be in the converter file?

Reply:
These will be used in both reconciler and converter.